src/sentry/api/base.py

```diff
@@ -241,7 +241,7 @@ def paginate(
         else:
             input_cursor = None
 
-        assert per_page <= max(100, default_per_page)
+        assert per_page <= max(1000, default_per_page)
```

Comment:
@macqueen - do you think this change is okay?

Reply:
we could let you customize the max per endpoint but i think generically allowing people to return 1k results on any endpoint might have consequences

Reply:
Yeah that's fair, let's pass a max value into paginate with a default of 100